R/info.R

```diff
@@ -0,0 +1,32 @@
+#' Report information of the package
+#'
+#' @return A list with information of the package
+#' @examples
+#' polars_info()
+#' @export
+polars_info = function() {
+  # Similar to arrow::arrow_info()
+  out = list(
+    version = utils::packageVersion("polars"),
+    features = FeatureInfo$new()$to_r()
+  )
+
+  structure(out, class = "polars_info")
+}
+
+#' @export
+print.polars_info = function(x, ...) {
+  # Copied from the arrow package
+  # https://github.com/apache/arrow/blob/6f3bd2524c2abe3a4a278fc1c62fc5c49b56cab3/r/R/arrow-info.R#L149-L157
+  print_key_values = function(title, vals, ...) {
+    df = data.frame(vals, ...)
+    names(df) = ""
+
+    cat(title, ":", sep = "")
+    print(df)
+    cat("\n")
+  }
+
+  cat("R Polars package version: ", format(x$version), "\n\n", sep = "")
```

Comment:
is it possible to have the rust-polars version as well? (and does it make sense to display it?)

Reply:
Good point.
But we uses unreleased version of rust-polars here, so I think showing rust-polars' version make little sense.

Reply:
Upon further investigation, it seems that there is no constant in rust-polars to display the version.
So I think we need to update rust-polars first to do that.

Reply:
I have opened an issue about that. pola-rs/polars#9659